Sources/VitalHealthKit/HealthKit/Models/VitalResource+HealthKit.swift

```diff
@@ -79,7 +79,8 @@ func toHealthKitTypes(resource: VitalResource) -> Set<HKObjectType> {
 
       return base + [
         HKSampleType.quantityType(forIdentifier: .heartRate)!,
-        HKSampleType.quantityType(forIdentifier: .restingHeartRate)!
+        HKSampleType.quantityType(forIdentifier: .restingHeartRate)!,
+        HKSampleType.quantityType(forIdentifier: .appleExerciseTime)!
```

Comment:
Please add this to `observedSampleTypes()`, so we would register to be notified of new `appleExerciseTIme` data as well.